src/main/scala/tilelink/Monitor.scala

```diff
@@ -31,30 +33,27 @@ object TLMonitor {
   }
 }
 
+@chiselName
 class TLMonitor(args: TLMonitorArgs, monitorDir: MonitorDirection = MonitorDirection.Monitor) extends TLMonitorBase(args)
 {
+  require (args.edge.params(TLMonitorStrictMode) || (! args.edge.params(TestplanTestType).formal))
+
   val cover_prop_class = PropertyClass.Default
-  val desc_text = "Placeholder"
-
-  /*def assert(cond: Bool, message: String): Unit = {
-    Property(monitorDir,
-        cond,
-        message,
-        PropertyClass.Default,
-        desc_text)
-  }*/
-
-  def assume(cond: Bool, message: String): Unit = if (monitorDir == MonitorDirection.Monitor)
-  {
+  val desc_text = "TLMonitor Assertion"
```

Comment:
desc_text is supposed to be for message, which should be set at each property instantiation.